test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -10,8 +10,18 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         if ($IsWindows) {
             $userName = "testuserservices"
-            $Password = ([char[]]([char]33..[char]95) + ([char[]]([char]97..[char]126)) + 0..9 | Sort-Object {Get-Random})[0..12] -join ''
-            $testPass = (New-Object -TypeName Net.NetworkCredential("", $Password)).SecurePassword
+            $numbers = "0123456789"
```

Comment:
Can you move this logic into a [HelpersCommons.psm1](https://github.com/PowerShell/PowerShell/blob/f645cb8bbbab13c447921d75d7e8dbfa6616a687/test/tools/Modules/HelpersCommon/HelpersCommon.psm1#L313)

Reply:
Sure